corehq/apps/accounting/management/commands/suppress_duplicate_invoices.py

```diff
@@ -0,0 +1,36 @@
+from django.core.management.base import BaseCommand
+from corehq.apps.accounting.models import Invoice
+from django.db.models import Count
+
+
+class Command(BaseCommand):
+    help = 'Suppress duplicate invoices based on a given start date'
+
+    def add_arguments(self, parser):
+        parser.add_argument('invoice_start_date', type=str,
+                            help='The start date of the invoices to be checked for duplicates')
+
+    def handle(self, *args, **kwargs):
+        invoice_start_date = kwargs['invoice_start_date']
+
+        # Filter the Invoice objects based on the date_start
+        invoices_of_given_date = Invoice.objects.filter(date_start=invoice_start_date)
+
+        # Annotate the filtered queryset with a count of each subscription id
+        duplicate_subscriptions = invoices_of_given_date.values('subscription').annotate(
+            count=Count('id')).filter(count__gt=1)
+
+        # Extract the subscription ids from the above queryset
+        duplicate_subs_ids = [item['subscription'] for item in duplicate_subscriptions]
+
+        suppressed_count = 0
+        for sub_id in duplicate_subs_ids:
+            related_invoices = Invoice.objects.filter(
+                subscription_id=sub_id, date_start=invoice_start_date).order_by('-date_created')
+            for invoice in related_invoices[1:]:
+                invoice.is_hidden_to_ops = True
+                invoice.save()
+                suppressed_count += 1
```

Comment:
In scripts like this is a good idea to provide detailed output of what actions were performed, which you can then copy and save as a record on the ticket.

So here that might be

```suggestion
                print(f"Suppressing invoice {invoice.id} ({invoice.subscription.subscriber.domain} ", end='')
                invoice.save()
                suppressed_count += 1
                print(f"✓")
```
or something like that.

Reply:
Thanks @dannyroberts ! Fixed here: 2aa271213171a8527169e10a9661ecb1a87139c9